docs/internals/howto-release-django.txt

```diff
@@ -97,33 +125,68 @@ You'll need a few things before getting started:
        rest_hostname = https://rest.api.transifex.com
        token = # API token
 
-* Access to the ``djangoproject.com`` server to upload files.
+* Access to the ``djangoproject.com`` server to upload files (using ``scp``).
 
-* Access to the admin on ``djangoproject.com`` as a "Site maintainer".
+* Access to the Django admin on ``djangoproject.com`` as a "Site maintainer".
 
-* Access to post to ``django-announce``.
+* Access to send emails to the following mailing lists and to create a post in
+  the `Django Forum - Announcements category
+  <https://forum.djangoproject.com/c/announcements/7>`_.
 
-* If this is a security release, access to the pre-notification distribution
-  list.
+  * `django-users <https://groups.google.com/g/django-users/>`_
+  * `django-developers <https://groups.google.com/g/django-developers/>`_
+  * `django-announce <https://groups.google.com/g/django-announce/>`_
 
-If this is your first release, you'll need to coordinate with another releaser
-to get all these things lined up.
+* Access to the ``django-security`` repo in GitHub. Among other things, this
+  provides access to the pre-notification distribution   list.
```

Comment:
```suggestion
  provides access to the pre-notification distribution list.
```
Also maybe we want to say "If this is a security release" 🤔 unsure if the "backup" releaser will always need this access but probably best for them to have this also (so maybe not required)